### DISCUSSION TITLE
Using with Grav cms?

### DISCUSSION THREAD

**Discussion by symgryph** [Pagefind/pagefind#439] [Q&A]
I suspect this doesn't work well with something like grav, which genreates its html pages from .md files, correct?

**Comment by bglw** [comment]
👋 @symgryph — converted this to a Discussion Q&A item so it's discoverable for others in the future! 🙂

I think for Grav, you might find some success with either [grav-plugin-blackhole](https://github.com/barryanders/grav-plugin-blackhole) or [grav-plugin-static-generator](https://github.com/OleVik/grav-plugin-static-generator) 

Both of those look like they can take a Grav site and spit out the static set of HTML files, which you could then index with Pagefind, adding Pagefind's index directory back to your site.

Let me know if you have a play in that direction and find any success!
